docs/gcp_create_new_project.md

```diff
@@ -0,0 +1,39 @@
+# using a new GCP project
+
+```
+# create a key from the project service account
+# upload it to our credential store
```

Comment:
would be good to link instructions how to upload to credential store.
I can't find one - so maybe create followup task?

Reply:
It's something I would rather not disclose in SCT open source code base...

Reply:
true...